src/connection_pool.rs

```diff
@@ -537,6 +535,30 @@ where
     }
 }
 
+impl<C, R, F> Deref for PoolConnection<C, R, F>
+where
+    C: Connection,
+    R: Resolver,
+    F: FnMut(&Backend) -> C + Send
+{
+    type Target = C;
+
+    fn deref(&self) -> &C {
+        &(self.connection_pair.0).1.as_ref().unwrap()
+    }
+}
+
+impl<C, R, F> DerefMut for PoolConnection<C, R, F>
+where
+    C: Connection,
+    R: Resolver,
+    F: FnMut(&Backend) -> C + Send
+{
+    fn deref_mut(&mut self) -> &mut C {
+        (self.connection_pair.0).1.as_mut().unwrap()
```

Comment:
Same here (panic). Maybe it's ok, just wondering?

Reply:
Same explanation applies to this one as the Deref case.